### PR TITLE
i18n(import): new WordPress links & small corrections

### PR DESCRIPTION
**Pull request by exterkamp**
**Summary**
`Automated` import of strings from translation. Mostly WordPress changes and some small corrections to a few langs.

### CODE REVIEW

File: lighthouse-core/lib/i18n/locales/pt-PT.json [hunk 1]
```diff
@@ -1581,13 +1581,13 @@
     "message": "Considere reduzir ou mudar o número de [plug-ins do WordPress](https://wordpress.org/plugins/) que carregam JavaScript não utilizado na sua página. Para identificar plug-ins que estejam a adicionar JS não reconhecido, experimente realizar a [cobertura de código](https://developers.google.com/web/updates/2017/04/devtools-release-notes#coverage) nas DevTools do Chrome. Pode identificar o tema/plug-in responsável a partir do URL do script. Esteja atento a plug-ins que tenham muitos scripts na lista com muito vermelho na cobertura do código. Um plug-in só deve ter um script na lista de espera se este for realmente utilizado na página."
   },
   "stack-packs/packs/wordpress.js | uses_long_cache_ttl": {
-    "message": "Leia sobre [Colocação do navegador em cache no WordPress](https://wordpress.org/support/article/optimization/#browser-caching)."
+    "message": "Leia sobre [Colocação do navegador em cache no WordPress](https://codex.wordpress.org/WordPress_Optimization#Browser_Caching)."
```
Comment (connorjclark) [inline lighthouse-core/lib/i18n/locales/pt-PT.json]: this page redirects you to the old url ...

Comment (exterkamp) [reply lighthouse-core/lib/i18n/locales/pt-PT.json]: Hm...Good call. I think something got messed up in the handoff, that is the old link.

Comment (exterkamp) [reply lighthouse-core/lib/i18n/locales/pt-PT.json]: :eyes: Paul https://github.com/GoogleChrome/lighthouse/pull/10069

**Comment by exterkamp** [comment]
Done by #10244

**Pull request closed without merging**